build.sbt

```diff
@@ -68,14 +76,18 @@ lazy val anorm = project
       // New functions
       missMeth("anorm.Sql.unsafeStatement$default$2"),
       missMeth("anorm.Sql.unsafeResultSet"),
-      missMeth("anorm.Sql.unsafeStatement")),
+      missMeth("anorm.Sql.unsafeStatement"),
+      missMeth("anorm.ToStatement.contramap"),
```

Comment:
I don't think that the case of a project changing the Anorm JARs without recompiling should be supported.

Anorm (and Play modules in general I think) never claims to have backward compatibility on the ABI.

Whereas it should for the API, which is ok for new functions.